### PR TITLE
[EASI-3691] Backend updates to geography question feature branch

### PR DESCRIPTION
**Pull request by OddTomBrooks**
# EASI-3691

## Changes and Description

- Added enums
- Added column for geographies region type
- Added column for geographies states and territories type
- Updated schema and DTOs

## How to test this change

- Bring server containers up
- Test endpoints
- Assert no errors and database values updated accordingly

## PR Author Review Checklist

- [x] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [x] Added or updated tests for backend resolvers or other functions as needed.
- [x] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [x] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: pkg/models/plan_general_characteristics.go [hunk 1]
```diff
@@ -76,3 +78,145 @@ func NewPlanGeneralCharacteristics(tls baseTaskListSection) *PlanGeneralCharacte
 		baseTaskListSection: tls,
 	}
 }
+
+/*
+enum StatesAndTerritories {
```
Comment (StevenWadeOddball) [inline pkg/models/plan_general_characteristics.go:83]: Can this comment be removed?

File: pkg/models/plan_general_characteristics.go [hunk 2]
```diff
@@ -76,3 +78,145 @@ func NewPlanGeneralCharacteristics(tls baseTaskListSection) *PlanGeneralCharacte
...
+  WY,
+  AS,
+  GU,
+  MP,
+  PR,
+  UM,
+  VI
+}
+
+enum GeographyRegionType {
```
Comment (StevenWadeOddball) [inline pkg/models/plan_general_characteristics.go:143]: Can this comment be removed?

**Review by StevenWadeOddball — CHANGES_REQUESTED** [review]
Thanks @OddTomBrooks , I tested and it looks good. Can you delete the old comments please in the models file please? I'll approve it as soon as they are removed.

File: pkg/models/plan_general_characteristics.go (hunk 1 above)
Comment (OddTomBrooks) [reply pkg/models/plan_general_characteristics.go:83]: Great catch, done.

**Review by StevenWadeOddball — APPROVED** [review]
🚀

**Pull request merged**